ecs-cli/modules/commands/compose/service/compose_service_command.go

```diff
@@ -189,3 +190,14 @@ func loadBalancerFlags() []cli.Flag {
 		},
 	}
 }
+
+// OptionalRegionFlag inline overrides region
+func ComposeServiceTimeOutFlag() cli.Flag {
```

Comment:
Yeah, it is added. Service down doesn't exist. Its a combination of stop and rm.